tests/e2e/stablehlo_models/BUILD.bazel

```diff
@@ -5,43 +5,31 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 # Tests for end-to-end IREE support of *simple* models or their close
-# derivatives. This does not use our standard test infrastructure and instead
-# JIT-compiles and runs models with iree-run-mlir. It is intended for small
-# example programs that can be manually read and updated.
+# derivatives.
 
+load("//build_tools/bazel:iree_check_test.bzl", "iree_check_test_suite")
 load("//build_tools/bazel:enforce_glob.bzl", "enforce_glob")
-load("//build_tools/bazel:iree_lit_test.bzl", "iree_lit_test_suite")
 
 package(
     features = ["layering_check"],
     licenses = ["notice"],  # Apache 2.0
 )
 
-iree_lit_test_suite(
-    name = "lit",
+iree_check_test_suite(
+    name = "check",
```

Comment:
Going to revert and redo some of these changes, per discussion [here on Discord](https://discord.com/channels/689900678990135345/690274711523164166/1202025787407204402). lit tests are useful, so we can keep a few of them but using llvm-cpu instead of vulkan/metal/etc.